db/migrate/20201117010549_add_optional_weekly_email_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddOptionalWeeklyEmailToUser < ActiveRecord::Migration[6.0]
+  def change
+    add_column :users, :weekly_email, :boolean, default: true
```

Comment:
would change the name to `notifications_enabled` or `email_notifications_enabled`